README.md

```diff
@@ -152,11 +152,13 @@ Measured::Weight.unit_names_with_aliases
   * oz, ounce, ounces
 * `Measured::Length`
   * m, meter, metre, meters, metres
+  * km, kilometer, kilometre, kilometers, kilometres
```

Comment:
Nitpick that I would have maybe an example, meaning that I don't think I'd grok what `all SI prefixes` means without an `ex: kilometer, centimeter, etc`